pkg/cloudprovider/metrics/metrics_suite_test.go

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
nit: Do you mind generically naming this `suite_test`. We are generally naming things `suite_test` as the top-level testing and then other tests propagate off of that.

Reply:
done